src/header/header.scss

```diff
@@ -11,9 +11,8 @@
   display: flex;
   justify-content: space-between;
   width: 100%;
-  height: $iui-baseline * 5 + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
+  height: $iui-baseline * 5 + $iui-xxs;
```

Comment:
Generally for changing from `content-box` to `border-box`, the height and width needs to increase to account for any padding or borders or anything that wasn't included in the height/width previously. So removing `+ $iui-xxs` made the header shorter, but I didn't need to add anything to keep it the same size. I am assuming that the height should stay the same rather than the being shorter? I will ask Jon about it when he gets back; I think he's the one that made the comment.